clients/hipfft_params.h

```diff
@@ -439,6 +488,13 @@ class hipfft_params : public fft_params
 
     void validate_fields() const override
     {
+        if(multiGPU > 1 && auto_allocate == fft_auto_allocation_off)
+        {
+            // hipfftXtSetWorkArea would be required
+            throw std::runtime_error(
+                "cannot request externally-managed work areas with multi-gpu usage");
+        }
```

Comment:
This isn't really an error; we just haven't implemented it yet.  Normally, one would just skip the test using gtest's mechanism, but we can't do that here because it's in the fft params level.